build.gradle.kts

```diff
@@ -59,6 +59,7 @@ allprojects {
             targetExclude("$buildDir/**/*.kt")
             targetExclude("bin/**/*.kt")
             ktlint(libs.versions.ktlint.get())
+                .editorConfigOverride(mapOf("ktlint_filename" to "disabled"))
```

Comment:
`ktlint_filename` is invalid, it should be `ktlint_standard_filename` in Ktlint 0.48.1, see https://pinterest.github.io/ktlint/faq/#why-is-editorconfig-property-disabled_rules-deprecated-and-how-do-i-resolve-this